From 94323d5dfc9b96b3f2845f41eba7d5e0cba35981 Mon Sep 17 00:00:00 2001
From: Emily Deng <Emily.Deng@amd.com>
Date: Thu, 14 Dec 2017 09:31:01 +0800
Subject: [PATCH 5722/5855] drm/amdkfd: KFD doesn't support TONGA SRIOV

KFD module doesn't support TONGA SRIOV, if init KFD module in TONGA SRIOV
environment, it will let compute ring IB test fail.

Signed-off-by: Emily Deng <Emily.Deng@amd.com>
Reviewed-by :Shaoyun.liu <Shaoyun.liu@amd.com>
---
 drivers/gpu/drm/amd/amdkfd/kfd_device.c | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_device.c b/drivers/gpu/drm/amd/amdkfd/kfd_device.c
index da8c718..74faa7d 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_device.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_device.c
@@ -101,21 +101,6 @@ static const struct kfd_device_info tonga_device_info = {
 	.num_sdma_engines = 2,
 };
 
-static const struct kfd_device_info tonga_vf_device_info = {
-	.asic_family = CHIP_TONGA,
-	.max_pasid_bits = 16,
-	.max_no_of_hqd  = 24,
-	.doorbell_size  = 4,
-	.ih_ring_entry_size = 4 * sizeof(uint32_t),
-	.event_interrupt_class = &event_interrupt_class_cik,
-	.num_of_watch_points = 4,
-	.mqd_size_aligned = MQD_SIZE_ALIGNED,
-	.is_need_iommu_device = false,
-	.supports_cwsr = false,
-	.needs_pci_atomics = false,
-	.num_sdma_engines = 2,
-};
-
 static const struct kfd_device_info fiji_device_info = {
 	.asic_family = CHIP_FIJI,
 	.max_pasid_bits = 16,
@@ -292,7 +277,6 @@ static const struct kfd_deviceid supported_devices[] = {
 	{ 0x6928, &tonga_device_info   },	/* Tonga */
 	{ 0x6929, &tonga_device_info   },	/* Tonga */
 	{ 0x692B, &tonga_device_info   },	/* Tonga */
-	{ 0x692F, &tonga_vf_device_info   },	/* Tonga vf */
 	{ 0x6938, &tonga_device_info   },	/* Tonga */
 	{ 0x6939, &tonga_device_info   },	/* Tonga */
 	{ 0x7300, &fiji_device_info    },	/* Fiji */
-- 
2.7.4

